inst/cpp/sim.cpp

```diff
@@ -126,7 +127,6 @@ List sim_wrapper_cpp (NumericVector A, List design, List par, NumericVector iov_
     ode_out tmp = sim_cpp(Aupd, t_start, t_end, step_size);
     if(start == 0) { // make sure observation variables are stored
       int k = 0;
-      // insert time-dependent covariates scale
```

Comment:
this is not needed. It was calculating the covariate values in the output table. No need to recompute.